supportedBackends/google/pipelines/batch/src/main/scala/cromwell/backend/google/pipelines/batch/GcpBatchAsyncBackendJobExecutionActor.scala

```diff
@@ -881,14 +875,24 @@ class GcpBatchAsyncBackendJobExecutionActor(override val standardParams: Standar
       _ <- uploadGcsDelocalizationScript(createParameters, gcsDelocalizationScriptCloudPath, transferLibraryContainerPath, gcsTransferConfiguration)
       //_ = this.hasDockerCredentials = createParameters.privateDockerKeyAndEncryptedToken.isDefined
       //_ <- uploadGcsTransferLibrary(createParameters, gcsTransferLibraryCloudPath, gcsTransferConfiguration)
-      _ = backendSingletonActor ! GcpBatchRequest(workflowId, createParameters, jobName = jobTemp, gcpBatchCommand, gcpBatchParameters)
-      runId = StandardAsyncJob(jobTemp)
-
-    }
-    yield runId
-
-    runBatchResponse map { runId => PendingExecutionHandle(jobDescriptor, runId, Option(Run(runId)), previousState = None) }
-
+      timeout = Timeout(60.seconds) // TODO: Alex - reconsider whether this is necessary, we could wait asynchronously for the job to get submitted
+      request = GcpBatchRequest(workflowId, createParameters, jobName = jobTemp, gcpBatchCommand, gcpBatchParameters)
+      response <- ask(backendSingletonActor, GcpBatchBackendSingletonActor.Action.SubmitJob(request))(timeout).map {
```

Comment:
Now, we use the ask pattern to get the job submission outcome (success/failure).

Later, we can remove the ask pattern to behave like google pipeline backend.